README.md

```diff
@@ -540,6 +540,18 @@ db.update({ _id: 'id6' }, { $push: { fruits: { $each: ['banana', 'orange'] } } }
 db.update({ _id: 'id6' }, { $push: { fruits: { $each: ['banana'], $slice: 2 } } }, {}, function () {
   // Now the fruits array is ['apple', 'orange']
 });
+
+// Update the value of the field, only if specified field is smaller or greater than the current value of the field
+// Let's say the database contains this document
+// doc = { _id: 'id1', name: 'Name', value: 5 }
+// $min can be used to update field if provided value is less than the current document value 
+db.update({ _id: 'id1' }, { $min: { value: 2 } }, {}, function () {
+  // The document will be updated to { _id: 'id1', name: 'Name', value: 2 }
+});
+// $max can be used to update field if provided value is greater than the current document value 
+db.update({ _id: 'id1' }, { $max: { value: 8 } }, {}, function () {
+  // The document will be updated to { _id: 'id1', name: 'Name', value: 8 }
+});
```

Comment:
Usually in the readme I prefer to show all behaviors of only one operator and let the reader understand that it works the same for the symetric operator. I changed this part to an example where `$min` doesn't modify the document.